build-tests/api-extractor-scenarios/etc/exportImportStarAs2/api-extractor-scenarios.api.json

```diff
@@ -190,14 +190,10 @@
                   "kind": "Content",
                   "text": "export declare function exportedApi(): "
                 },
-                {
-                  "kind": "Content",
-                  "text": "forgottenNs."
-                },
                 {
                   "kind": "Reference",
-                  "text": "ForgottenClass",
-                  "canonicalReference": "api-extractor-scenarios!~ForgottenClass:class"
+                  "text": "forgottenNs.ForgottenClass",
+                  "canonicalReference": "api-extractor-scenarios!ForgottenClass:class"
```

Comment:
This reference is incorrect, but I'm not sure what the right reference is (another instance of https://github.com/microsoft/rushstack/issues/1308). I'm thinking it should probably be `api-extractor-scenarios!~ForgottenClass:class`, and `api-extractor-scenarios!~ForgottenClass:class` is definitely _more correct_ that `"api-extractor-scenarios!ForgottenClass:class"` (as `ForgottenClass` is un-exported from the package, and the `~` indicates that).

I'll leave a comment above for how we can slightly modify `DeclarationReferenceGenerator` to fix this, then feel free to mark this as resolved. 👍